src/rooted_sbn_instance.hpp

```diff
@@ -185,7 +185,10 @@ TEST_CASE("RootedSBNInstance: subsplit support and TrainSimpleAverage") {
                                          {"00010|00101|00001", 1},
                                          {"00001|01110|00100", 1},
                                          {"00010|11101|00100", 1}});
-  CHECK_EQ(correct_parameters, inst.PrettyIndexedSBNParameters());
+  sort(correct_parameters.begin(), correct_parameters.end());
```

Comment:
How does this compile with `sort` vs `std::sort`? Did we open `std` somewhere?

Reply:
It is using argument dependent lookup to find `sort` in `std`. I changed `sort` to `std::sort` to make it explicit.